src/components/calcite-card/calcite-card.scss

```diff
@@ -8,7 +8,6 @@
       bg-foreground-1
       duration-150
       ease-in-out
-      relative
```

Comment:
@y0n4 the PR title and commit message is for dropdown but there are no changes dropdown, only card.

Can we update the PR title/description?

Does this need a test of some kind?